content/06.methods.md

```diff
@@ -243,15 +243,16 @@ We aimed to define focal copy number units to avoid calling adjacent genes in th
 To determine the most focal units, we first considered the dominant status calls at the chromosome arm-level.
 If the chromosome arm dominant status was callable but not clearly defined as a gain or loss, we instead included the cytoband-level status call.
 Similarly, if a cytoband dominant status call was callable but not clearly defined as a gain or loss, we instead included gene-level status call.
-To obtain the gene-level data, we used the `mergeByOverlaps` [function](https://www.rdocumentation.org/packages/IRanges/versions/2.6.1/topics/findOverlaps-methods) from the IRanges package in R [@doi:10.1371/journal.pcbi.1003118] to find overlaps between the segments in the consensus CNV file and the exons in the GENCODE v27 annotation file (See **Key Resources Table**)	.
+To obtain the gene-level data, we used the `mergeByOverlaps` [function](https://www.rdocumentation.org/packages/IRanges/versions/2.6.1/topics/findOverlaps-methods) from the `IRanges` package in R [@doi:10.1371/journal.pcbi.1003118] to find overlaps between the segments in the consensus CNV file and the exons in the GENCODE v27 annotation file (See **Key Resources Table**)	.
 If the copy number value was 0, we set the status to "deep deletion". 
 For autosomes only, we set the status to "amplification" when the copy number value was greater than two times the ploidy value.
+We plotted genome-wide gains and losses in (**Figure {@fig:S3}B**) using the R package `ComplexHeatmap` [@doi:10.1093/bioinformatics/btw313].
```

Comment:
What if we put these citations directly in the relevant captions? Would that save space? Like, "Panel B was made with `ComplexHeatmap` [@doi:10.1093/bioinformatics/btw313]," and similar.

Reply:
Methods doesn't count towards space, but I think figure captions may.. 

>* Resources should be under 45,000 characters (including spaces but not including the reference list, STAR Methods, or supplemental information). There may be up to 7 display items. Additional display items and materials may be published as supplemental information.

So it looks like legends may count. I have no idea what our count is now, so maybe we can assess once it is in the doc?